tests/test_view.py

```diff
@@ -0,0 +1,28 @@
+"""Unit tests for view.py"""
+
+#import pytest
+from todo_app.view import ViewModel
+
+def test_viewmodel_empty_filterToDo_only():
+    emptyViewModel = ViewModel([])
+    assert len(emptyViewModel.todo) == 0
+
+def test_viewmodel_filterTodo_only():
+    view_model = ViewModel([{"title":"Task1", "due date":"2021-03-25", "status":"Not Started"}])
+    assert view_model.items[0]['status'] == "Not Started"
```

Comment:
To clarify, this test (and test_viewmodel_filterDoing_only and test_viewmodel_filterDone_only) are currently pointless. They are actually all doing the same thing.

The test name claims to be checking the filtering works, but it doesn't - it gets the whole list (which happens to contain only one item, with status Not Started) and then asserts that the list still includes the one item.

Also you should create a viewmodel with multiple items with various statuses, and then assert that view_model.todo only contains the expected ones. Otherwise the tests would still pass if the properties didn't bother with any filtering.

Reply:
Thank you for feedback. I see where I have gone wrong and now corrected.